InterfaceTPL/tsgBlasWrappers.hpp

```diff
@@ -73,6 +73,11 @@ void dgels_(const char *trans, const int *M, const int *N, const int *nrhs, doub
             double *B, const int *ldb, double *work, int *lwork, int *info);
 void zgels_(const char *trans, const int *M, const int *N, const int *nrhs, std::complex<double> *A, const int *lda,
             std::complex<double> *B, const int *ldb, std::complex<double> *work, int *lwork, int *info);
+// Symmetric tridiagonal eigenvalue compute
+void dstebz_(const char *range, const char *order, const int *N, const double *vl, const double *vu, const int *il, const int *iu, const double *abstol,
+             const double D[], const double E[], int *M, int *nsplit, double W[], int iblock[], int isplit[], double work[], int iwork[], int *info);
```

Comment:
In an earlier comment you mentioned that `E` is destroyed during the call. Is that so? If so, then this should not be `const double E[]`. Double check the other arrays.

Reply:
Only ``dbsterf`` and ``dsteqr`` destroy ``E``. The function ``dstebz``, on the other hand, outputs the eigenvalues to another array ``W``.

For reference, see:
http://www.netlib.org/lapack/explore-html/d2/d24/group__aux_o_t_h_e_rcomputational_gaf0616552c11358ae8298d0ac18ac023c.html
http://www.netlib.org/lapack/explore-html/d2/d24/group__aux_o_t_h_e_rcomputational_ga47fe470e7a882c58d4cc49e2c6cb7c70.html
https://www.netlib.org/lapack/explore-html/d2/d24/group__aux_o_t_h_e_rcomputational_ga28f88843da09a0ee400daf46caaabec6.html

Reply:
OK, so long as everything is consistent.